website/content/docs/platform/k8s/vso/sources/vault.mdx

```diff
@@ -0,0 +1,283 @@
+spec:
+  vaultAuthRef: vault-auth
+  mount: pki
+  role: default
+  commonName: example.com
+  format: pem
+  expiryOffset: 1s
+  ttl: 60s
+  namespace: tenant-1
+  destination:
+    create: true
+    name: pki1
+```
+
+### VaultDynamicSecret custom resource
+
+Provides the configuration necessary for the Operator to synchronize a single Vault *dynamic* Secret to a single Kubernetes Secret.<br />
+Supported secrets engines *non-exhaustive*: [databases](/vault/docs/secrets/databases), [aws](/vault/docs/secrets/aws),
+[azure](/vault/docs/secrets/azure), [gcp](/vault/docs/secrets/gcp), ...
+
+##### Database secret example
+
+The database secret engine's `db` mount path is specified under `spec.mount` of `VaultDynamicSecret` custom resource. Please consult [Database Secrets Engine - Setup](/vault/docs/secrets/databases#setup) for configuring the database secrets engine. The following results in a request to `http://127.0.0.1:8200/v1/db/creds/my-postgresql-role` to generate a new credential.
+
+```yaml
+---
+apiVersion: secrets.hashicorp.com/v1beta1
+kind: VaultDynamicSecret
+metadata:
+  namespace: vso-example
+  name: vault-dynamic-secret-db
+spec:
+  vaultAuthRef: vault-auth
+  mount: db
+  path: creds/my-postgresql-role
+  destination:
+    create: true
+    name: dynamic-db
+```
+
+##### AWS secret example
+
+The AWS secrets engine's `aws` mount path is specified under `spec.mount` of `VaultDynamicSecret` custom resource. Please consult [AWS Secrets Engine - Setup](/vault/docs/secrets/aws#setup) for configuring AWS secrets engine.
+
+The following results in a request to `http://127.0.0.1:8200/v1/aws/creds/my-iam-role` to generate a new IAM credential.
+
+```yaml
+---
+apiVersion: secrets.hashicorp.com/v1beta1
+kind: VaultDynamicSecret
+metadata:
+  namespace: vso-example
+  name: vault-dynamic-secret-aws-iam
+spec:
+  vaultAuthRef: vault-auth
+  mount: aws
+  path: creds/my-iam-role
+  destination:
+    create: true
+    name: dynamic-aws-iam
+```
+
+The following results in a request to `http://127.0.0.1:8200/v1/aws/sts/my-sts-role` to generate a new STS credential.
+
+```yaml
+---
+apiVersion: secrets.hashicorp.com/v1beta1
+kind: VaultDynamicSecret
+metadata:
+  namespace: vso-example
+  name: vault-dynamic-secret-aws-sts
+spec:
+  vaultAuthRef: vault-auth
+  mount: aws
+  path: sts/my-sts-role
+  destination:
+    create: true
+    name: dynamic-aws-sts
+```
+
+##### HCP Vault Secrets Example
+
+
+```yaml
+---
+apiVersion: secrets.hashicorp.com/v1beta1
+kind: VaultDynamicSecret
+metadata:
+  namespace: vso-example
+  name: vault-dynamic-secret-aws-iam-role
+spec:
+  vaultAuthRef: vault-auth
+  mount: aws
+  path: creds/my-iam-role
+  destination:
+    create: true
+    name: dynamic-aws-iam-role
+```
+
+@include 'vso/blurb-api-reference.mdx'
+
+## Tutorial
+
+Refer to the [Vault Secrets Operator on
+Kubernetes](/vault/tutorials/kubernetes/vault-secrets-operator) tutorial to
+learn the end-to-end workflow using the Vault Secrets Operator.
```

Comment:
```suggestion
Refer to the [Vault secrets operator on
Kubernetes](/vault/tutorials/kubernetes/vault-secrets-operator) tutorial to
learn the end-to-end workflow using the Vault secrets operator.
```

Reply:
The suggestion seems good, except it the product name is not properly capitalized. It should be "Vault Secrets Operator"